File: programmers.co.kr/행렬의 덧셈.py
def solution(arr1, arr2):
    new_arr = []
    for i in range(len(arr1)):
        arr = []
        for j in range(len(arr1[i])):
            arr.append(arr1[i][j] + arr2[i][j])
        new_arr.append(arr)
    return new_arr

File: programmers.co.kr/행렬의 덧셈.js
function solution(arr1, arr2) {
    var answer = [];
    
    for (var i = 0; i < arr1.length; i++){
        var a = arr1[i];
        var b = arr2[i];
        let c = new Array();
        for (var j = 0; j < a.length; j++){
            var d = a[j] + b[j];
            c.push(d);
        }
        answer.push(c);
    }
    return answer;
}